moduleframework/dockerlinter.py

```diff
@@ -236,10 +236,16 @@ def check_helpmd_is_present(self):
                  False if help.md is not specified in Dockerfile
         """
         helpmd_present = False
-        for c in self.docker_dict[COPY]:
-            if "help.md" in c:
-                helpmd_present = True
-        for a in self.docker_dict[ADD]:
-            if "help.md" in a:
-                helpmd_present = True
+        try:
+            for c in self.docker_dict[COPY]:
+                if "help.md" in c:
+                    helpmd_present = True
+        except KeyError:
+            print("Tag COPY is not present in Dockerfile")
```

Comment:
It's not 'tag', it's an instruction: https://docs.docker.com/engine/reference/builder/